x-pack/test/reporting/es_archives/historic/mappings.json

```diff
@@ -82,6 +82,13 @@
             }
           }
         },
+        "file-upload-telemetry": {
```

Comment:
Why was this added?

Reply:
Just removed it. I had a failing test that was leveraging `es_archives` and briefly speculated their mapping would also require an update.